plugins/prebuffer-mixin/src/ffmpeg-rebroadcast.ts

```diff
@@ -134,6 +134,10 @@ export async function startParserSession<T extends string>(ffmpegInput: FFmpegIn
 
 
     const args = ffmpegInput.inputArguments.slice();
+    const env = {
```

Comment:
I mean this should only be set if ffmpegInput.env is used, otherwise use undefined, which is the existing behavior.

Reply:
env = ffmpegInput.env ? { ...process.env, ...ffmpegInput.env } : undefined;